### PR TITLE
Update openai

### PR DESCRIPTION
**Pull request by yfw918**
Add the domain for GPT4 uploading files.

### CODE REVIEW

**Comment by FDUZS** [comment]
Please include `chatgpt.com` as well. I'm unsure why PR #1899 was closed.

**Comment by yfw918** [comment]
> Please include `chatgpt.com` as well. I'm unsure why PR #1899 was closed.

I added it.

**Comment by FDUZS** [comment]
Thank you. You may need to alphabetically sort those domains.

File: data/openai [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Main domain
 openai.com
 oaistatic.com
+chatgpt.com
+oaiusercontent.com
```
Comment (Loyalsoldier) [inline data/openai:5]:
```suggestion
chatgpt.com
oaistatic.com
oaiusercontent.com
openai.com
```

**Pull request merged**